src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,70 @@
+package step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator{
+
+    public static final int ZERO = 0;
+    public static final String DELIMITERS = ",|:";
+
+    public static int splitAndSum(String text){
+        //빈문자열 또는 null을 입력한 경우
+        if(isBlank(text)){return ZERO;}
+
+        return sum(toInts(split(text)));
+
+        /*if(text.length() > 1) {
+            //구분자를 컴마(,) 이외에 콜론(:)을 사용한 경우
+            if(text.contains(":")){
+                String[] tokens = text.split(":|,");
+                int result = 0;
+                for(int i=0; i < tokens.length; i++) {
+                    result += Integer.parseInt(tokens[i]);
+                }
+                return result;
+            }
+            //숫자 두개를 컴마(,) 구분자로 입력한 경우
+            if(text.contains(",")){
+                String[] numbers = text.split(",");
+                return Integer.parseInt(numbers[0]) + Integer.parseInt(numbers[1]);
+            }
+        }
+        */
+    }
+
+    private static int sum(int[] numbers) {
+        int sum = ZERO;
+        for(int number : numbers){
+            sum += number;
+        }
+        return sum;
+    }
+
+    private static String[] split(String text) {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(text);
```

Comment:
정규표현식은 언제나 헷갈리는 포인트인것 같아요.

그런면에서, 팀원이 보아도 쉽게 어떠한 의미를 가지는 표현식인지 알수 있도록 네이밍을 지어보는건 어떨까요?
```suggestion
        어떤패턴 = Pattern.compile("//(.)\n(.*)");
        Matcher 명확한의미 = 어떤패턴.matcher(text);
```